demo/src/app/pages/modules/range/range.page.html

```diff
@@ -0,0 +1,43 @@
+<demo-page-title>
+    <div header>Rating</div>
+    <div sub-header>
+        <p>A rating indicates user interest in content</p>
+    </div>
+</demo-page-title>
+<demo-page-content>
+    <sui-message class="info" [isDismissable]="false">
+        <div class="header">Important Note</div>
+        <p>
+            The range is a port of <a href="https://github.com/tyleryasaka/semantic-ui-range">Semantic-UI-Range</a>,
+            an external module for Semantic UI. If you'd like to use the range in your app, you must include the module's
+            CSS alongside Semantic UI's.
+        </p>
+        <div class="ui segment">
+            <demo-codeblock language="markup" [src]="cssInclude"></demo-codeblock>
```

Comment:
[This repo](https://github.com/gdaunton/Semantic-UI/tree/range-slider-build) has the code in it... I wonder if there is a way of easily pulling out the theming files?